controllers/topics/index.js

```diff
@@ -199,6 +199,10 @@ const getFollowerList = async (req, res) => {
             WHEN '' THEN false
             ELSE true
         END as following,
+        CASE COALESCE(is_following_subquery.user_id_follower, '')
+            WHEN '' THEN false
+            ELSE true
+        END as is_following,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for setting the `is_following` flag seems correct. However, it's important to ensure that `is_following_subquery.user_id_follower` is indeed a string and not a number or any other data type. If it's not a string, comparing it with an empty string ('') might lead to unexpected results. 

If `is_following_subquery.user_id_follower` is a number, you could modify the code as follows:

```diff
-        CASE COALESCE(is_following_subquery.user_id_follower, '')
-            WHEN '' THEN false
+        CASE WHEN is_following_subquery.user_id_follower IS NULL THEN false
             ELSE true
         END as is_following,
```

This change ensures that the `is_following` flag is set to `false` when `is_following_subquery.user_id_follower` is `NULL`, and `true` otherwise.


<!-- This is an auto-generated comment by OSS CodeRabbit -->